src/test/java/seedu/address/model/patient/BirthdayTest.java

```diff
@@ -30,10 +32,13 @@ public void isValidBirthdate() {
         assertFalse(Birthday.isValidBirthdate("january")); // non-numeric
         assertFalse(Birthday.isValidBirthdate("23071991")); // not in the format of dd/mm/yyyy
         assertFalse(Birthday.isValidBirthdate("22 /01 / 1997")); // spaces within digits
+        assertFalse(Birthday.isValidBirthdate("01 /01 / 3000")); // date in the future
```

Comment:
Perhaps the spaces within digits should be removed, in order to ensure that the assertion is false due to the date itself and not the spaces?

Reply:
Oh, I didn't notice that, thanks for pointing it out!